pkg/kubefed2/federate/enable.go

```diff
@@ -253,6 +253,10 @@ func typeConfigForTarget(apiResource metav1.APIResource, enableTypeDirective *En
 	spec := enableTypeDirective.Spec
 	kind := apiResource.Kind
 	pluralName := apiResource.Name
+	shortNames := []string{}
```

Comment:
This function is only responsible for creating a type config, and short names don't belong in the type config.  Please relocate this logic to the `GetResources` function and pass the computed short names in to `federateTypeCRD`.